### PR TITLE
Is ms reactlinks

### PR DESCRIPTION
**Pull request by Ginger-Mano**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <BrowserRouter>
+          <Navbar />
```
Comment (arvinf07) [inline src/App.js:14]: I think the links are supposed to be at the bottom.

"Links are present and persistent at the bottom of the app: one for the “list” view, the other for the “add an item” view"

Comment (arvinf07) [reply src/App.js:14]: I just saw the notes. Maybe this is not as easy as it seems

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  const [isOpen, setOpen] = useState(false);
+  return (
+    <nav
+      className="navbar is-primary"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">{/* ... */}</div>
+
+      <div className={`navbar-menu ${isOpen && 'is-active'}`}>
```
Comment (arvinf07) [inline src/components/Navbar.js:14]: Could you explain to me what this does?

File: src/App.js (hunk 1 above)
Comment (Ginger-Mano) [reply src/App.js:14]: I was able to get the links to the bottom, but commented out the code in the App.css file. If necessary, I can move it to the bottom!

**Pull request merged**